src/model/enums/misc.rs

```diff
@@ -0,0 +1,105 @@
+use serde::{Deserialize, Serialize};
+use strum::{AsRefStr, Display, EnumString};
+
+/// disallow: interrupting_playback, pausing, resuming, seeking, skipping_next, skipping_prev, toggling_repeat_context, toggling_shuffle, toggling_repeat_track, transferring_playback
```

Comment:
What is the `disallow: ...` part in the comment?

Reply:
｀disallow｀ just means `disallow key:`, I would rename it to `disallow key`.

Reply:
I still don't quite understand it, sorry. What's that for?

Reply:
Disallow means the [disallows object](https://developer.spotify.com/documentation/web-api/reference/object-model/#disallows-object), which is used in the endpoint [`get-users-currently-playing-track`](https://developer.spotify.com/documentation/web-api/reference/player/get-the-users-currently-playing-track/), is is clear to you now?

Reply:
Ahhh okay, yes, sorry. The link to the reference helped. For some reason I thought it was something for the linter.

Can you add the references for these structs/enums like the endpoints? They are usually like this:

```rust
/// Endpoint: description
///
/// ...
///
/// [Reference](reference)
```

just for consistency

Reply:
Yep, I have already updated this. This comments is outdated.

Reply:
Great. Can you make it 80 characters long as always?

Reply:
Yes, of course. I think it's a better way to let `rustfmt` handle it :)

Reply:
rustfmt doesn't modify comments AFAIK